examples/video_understanding/configs/workers/task_conqueror.yml

```
@@ -0,0 +1,5 @@
+name: TaskConqueror
+llm: ${sub|json_res}
+tool_manager: ${sub|all_tools}
+output_parser: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)